src/panels/CreateClusterPanel.ts

```diff
@@ -421,3 +439,10 @@ function isRestError(ex: unknown): ex is RestError {
 function isDefaultK8sVersion(version: KubernetesVersion): boolean {
     return "isDefault" in version && version.isDefault === true;
 }
+function getServicePrincipalId(result: AzureAuthenticationSession): string {
```

Comment:
💡 Key note please: this only get and I am guessing we are not creating any service principal on the fly? also the sample below if is nothing active and well randomised. :)

Reply:
we are not just getting the id from already authenticated azure session.

Reply:
removed the sample value.